app/classifier/components/Intervention/Intervention.spec.js

```diff
@@ -17,20 +17,31 @@ describe('Intervention', function () {
   };
   const onUnmount = sinon.stub()
 
-  before(function () {
-    wrapper = mount(
-      <Intervention
-        intervention={intervention}
-        onUnmount={onUnmount}
-        user={user}
-      />);
-  });
+  let wrapper = mount(
+    <Intervention
+      intervention={intervention}
+      onUnmount={onUnmount}
+      user={user}
+    />);
+
   it('should render', function () {
     expect(wrapper).to.be.ok;
   });
   it('should show a notification message', () => {
     const newlineMsg = intervention.message + "\n"
-    expect(wrapper.find(Markdown).text()).to.equal(newlineMsg);
+    expect(wrapper.find(Markdown).first().text()).to.equal(newlineMsg);
+  });
+  it('should show an opt-out message', () => {
+    const optOutHTML = '<div class="markdown "><p>I do not want to take part in this messaging ' +
+      '<a href="https://docs.google.com/document/d/1gLyN6Dgff8dOCOC88f47OD6TtFrfSJltsLgJMKkYMso/preview" ' +
+      'target="_blank" ref="noopener nofollow">study</a>. Do not show me further messages.</p>' +
+      '\n' + '</div>'
+    expect(wrapper.find(Markdown).last().html()).to.equal(optOutHTML);
+  });
+  it('should show a fallback to english opt-out message', () => {
```

Comment:
This is testing counterpart, not the component.

Reply:
The test was on the wrapper component output, which relies on counterpart. Sure i had to modify counterpart in setup...anyway i can't reliably get the test to fail as counterpart seems to have fallbackLocales setup by default. 

I couldn't see where this config was being set, default? So I'm going to assume that this is normal, expected behaviour and remove the test.

```
{ locale: 'en',
     interpolate: true,
     fallbackLocales: [ 'en' ],
     scope: null,
     translations: 
      { en: [Object],
        cs: [Object],
        it: [Object],
        es: [Object],
        nl: [Object],
        pt: [Object] },
     interpolations: {},
     normalizedKeys: { '.': [Object] },
     separator: '.',
     keepTrailingDot: false,
     keyTransformer: [Function: keyTransformer],
     generateMissingEntry: [Function: generateMissingEntry] },
```

Reply:
From memory, [counterpart falls back to English by default](https://www.npmjs.com/package/counterpart#locales). We should probably update that to set the default locale to the project primary language when a project loads, if we don't do that already. It just so happens that counterpart's default is the same as Panoptes' default, so it magically works.

Reply:
Ignore that. We set the fallback when the JS first loads.
https://github.com/zooniverse/Panoptes-Front-End/blob/9461eb8f83b4b12e0b9bee501cbe8d535783e1a7/app/redux/ducks/translations.js#L5-L11

Reply:
I commented out the two `counterpart.setFallbackLocale();` functions and it still had the fallback set. I was a bit lost at that point and decided `countepart` was magical